flowey/flowey_cli/src/cli/exec_snippet.rs

```diff
@@ -370,7 +370,7 @@ pub(crate) mod serialized_request {
     use serde::Deserializer;
     use serde::Serializer;
 
-    #[allow(clippy::borrowed_box)] // required by serde
+    #[expect(clippy::borrowed_box, reason = "Required by serde")]
```

Comment:
nit: unless there's a strong existing convention in the community to capitalize, I'd default to keeping these reason strings lowercase